l2tdevtools/dependency_writers/gift_ppa.py

```diff
@@ -72,35 +68,47 @@ class GIFTPPAInstallScriptWriter(interface.DependencyFileWriter):
       'fi',
       '']  # yapf: disable
 
-  def Write(self):
-    """Writes a gift_ppa_install.sh file."""
+  def _Write(self, python_dependencies, python_version_string='PYTHON2'):
+    """Writes a gift_ppa_install.sh file.
+
+    Args:
+      python_dependencies (list[str]): list of names of python dependencies.
+      python_version_string (Optional[str]): Python version.
```

Comment:
"Python version" this does not reflect the actual argument, please specify which values are passed here.

Also why `PYTHON2` and not just `2`? This also deviates from other parts of the codebase which uses `python_version`? I opt to keep the codebase consistent and use `python_version`.

Reply:
done